frontend/src/scenes/Routing/index.tsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Resources from '../Resources/index';
 import { Routes } from '../../common/enums/Routes';
+import NotFound from '../NotFound/components/NotFound';
 
 const ResourceRouting = () => (
   <Switch>
-    <Route path={Routes.Resources} component={Resources} />
-    <Route path={Routes.ResourcesEdit} component={() => <div>edit</div>} />
+    <Route exact path={Routes.Resources} component={Resources} />
+    <Route exact path={Routes.ResourcesEdit} component={() => <div>edit</div>} />
+    <Route path="*" component={NotFound} />
```

Comment:
we don't need it here

Reply:
OK.  I'll remove NotFound from this route.